components/payment_form/payment_form.tsx

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {getName} from 'country-list';
+import {FormattedMessage} from 'react-intl';
+
+import {PaymentMethod} from 'mattermost-redux/types/cloud';
+
+import {BillingDetails} from 'types/cloud/sku';
+
+import DropdownInput from 'components/dropdown_input';
+import Input from 'components/input';
+import * as Utils from 'utils/utils';
+import {COUNTRIES} from 'utils/countries';
+
+import StateSelector from './state_selector';
+import CardInput, {CardInputType} from './card_input';
+import CardImage from './card_image';
+
+import './payment_form.scss';
+import 'components/input.css';
+
+type Props = {
+    className: string;
+    initialBillingDetails?: BillingDetails | null;
```

Comment:
Since, this is an optional parameter, the type evaluates to `BillingDetails | null | undefined`.
For simplicity, could we remove the null and just check for `undefined` or just do `!this.props.initialBillingDetails`